crds/run_v1beta1_runjob.yaml

```diff
@@ -738,6 +738,35 @@ spec:
                         description: VPC Access configuration to use for this Task.
                           For more information, visit https://cloud.google.com/run/docs/configuring/connecting-vpc.
                         properties:
+                          connectorRef:
```

Comment:
Could you please revert changes in this file?

The files under `crds/` get updated automatically during each new release.

Reply:
Done, reverted the changes in the this commit: https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1085/commits/bc2e8125bd1d7ef63e7aced3beaf0b40ea52a67b